Det-store-nissehack/det-store-nissehack-ondsindede-loginforsøg/index.md

```diff
@@ -0,0 +1,138 @@
++++
+title = 'Det store nissehack: Ondsindede Loginforsøg'
+categories = ['Det store nissehack']
```

Comment:
This category...
I follow your choice, but... How about "Forensics" as the cateogry and then consider hiphening the title with the parts, so in full it will read:
Forensics: Det store nissehack - Onsindede Loginforsøg
Just as a suggestion as we can then reuse the category for future CTFs!

Reply:
Jeg har bare taget udgangspunkt i den kategori de var i hos NC3:
![image](https://github.com/user-attachments/assets/75ec1e2e-d817-43a0-8ada-407df2362956)

For min skyld kan vi sagtens skifte kategorierne til noget mere sigende 🤷